backend/__tests__/routes/doctors.test.js

```diff
@@ -112,8 +113,37 @@ describe("PATCH: update the priority of a patient as a doctor", () => {
         const data = {
             accountId: 51,
             patiendId: 3,
-            isPrioritized: constants.BOOLEANS.TRUE,
+            isPrioritized: BOOLEANS.TRUE,
         };
         await request(app).patch("/doctors/updatePriority").send(data).expect(401);
     });
 });
+
+describe("POST: makeAppointment - Adding a new appointment", () => {
+    it("Returns code 200 if appointment has been created successfully", async () => {
+        const testAppointment = {
+            patientId: TEST_CONSTANTS.APPOINTMENT_PATIENT_ACCOUNT.PatientId,
+            date: "2022-03-03",
+            time: "9:00 - 9:30",
+        };
+        await request(app).post("/doctors/makeAppointment/109").send(testAppointment).expect(200);
+        await Appointment.destroy({ where: { Patient_PatientId: testAppointment.patientId } });
+    });
+
+    it("Returns code 404 if user does not exist", async () => {
+        const testAppointment = {
+            patientId: 23617826387126,
+            date: "2022-03-03",
+            time: "9:00 - 9:30",
+        };
+        await request(app).post("/doctors/makeAppointment/109").send(testAppointment).expect(404);
+    });
+});
+
+describe("GET: getAppointmentsAndPatients", () => {
+    it("Returns code 200 and object of appointments & patients if user is a doctor", async () => {
+        await request(app)
+            .get(`/doctors/getAppointmentsAndPatients/${TEST_CONSTANTS.DOCTOR_ACCOUNT.AccountId}`)
+            .expect(200);
```

Comment:
I think you should test the object/data returned to see if it has the proper keys and values inside